src/common/UnitTests-CommonLib/Settings.Tests.cpp

```diff
@@ -101,19 +102,13 @@ namespace UnitTestsCommonLib
 
         TEST_METHOD (LoadFromEmptyString)
         {
-            auto func = [] { PowerToyValues values = PowerToyValues::from_json_string(L""); };
-            Assert::ExpectException<winrt::hresult_error>(func);
-        }
-
-        TEST_METHOD (LoadFromInvalidString_NameMissed)
```

Comment:
`from_json_string` does not use name property from the json string to set `result._key`. Hence the test would not throw any exception. I think it might be better to modify `from_json_string` function to explicitly check for the name property and throw error if that is empty. We can then use this test. What do you think?

Reply:
I think that sounds like a good idea.

Reply:
Great! I will update this.